src/ns_tracker/core.clj

```diff
@@ -1,21 +1,26 @@
 (ns ns-tracker.core
   "Keeps track of which namespaces have changed and need to be reloaded."
-  (:import java.io.PushbackReader)
+  (:import java.io.PushbackReader
+           java.io.File)
   (:require [clojure.java.io :as io])
   (:use [ns-tracker.dependency :only (graph seq-union depend dependents remove-key)]
         [ns-tracker.nsdeps :only (deps-from-ns-decl)]
         [ns-tracker.parse :only (read-in-ns-decl)]
         [clojure.java.io :only (file)]
-        [clojure.tools.namespace.find :only (find-clojure-sources-in-dir)]
+        [clojure.set :only (union)]
```

Comment:
Although this library is old and still uses `:use`, going forward lets be more idiomatic and prefer `:require :as`.